src/api/bkuser_core/profiles/migrations/0022_auto_20220517_1726.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.5 on 2022-05-17 09:26
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('profiles', '0021_auto_20210326_1640'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='profile',
+            name='account_expiration_time',
+            field=models.DateField(blank=True, null=True, verbose_name='账号过期时间'),
+        ),
+        migrations.AlterField(
+            model_name='profile',
+            name='category_id',
+            field=models.IntegerField(blank=True, null=True, verbose_name='用户目录ID'),
+        ),
+        migrations.AlterIndexTogether(
+            name='profile',
+            index_together={('category_id', 'account_expiration_time')},
```

Comment:
`account_expiration_time` to `account_expiration_date`?

Reply:
Done